### ISSUE TITLE
Secure transport between nodes using TLS

### ISSUE THREAD

**Issue by Enet4** [Enet4/dicom-rs#283]
The `ul` crate should provide an intuitive way to support securing a connection between two DICOM entities through TLS.

**Comment by pgimeno4d** [comment]
What's missing after #679?

**Comment by Enet4** [comment]
You're right, there is now baseline support for TLS. Feel free to file new issues for extensions and bug tracking of this capability.
